### DISCUSSION TITLE
Merhabalar

### DISCUSSION THREAD

**Discussion by 4rchin** [Ludeon/RimWorld-Turkish#320] [Q&A] [answered]
Yumeshi Novel Fansub Çevirmeni olarak projeye dahil oldum, Github'a aşina olmadığım için zorluk çekiyorum. Yardımcı olabilirseniz seve seve RimWorld'ü %100 Türkçe hale getirmeye gayret edeceğim. 

4rchain#1523 Discord profilimdir.

**Comment by LadySmith** [comment, marked as answer]
Discord üzerinden size yardımcı olabilirim. LadySmith#0763 üzerinden size istek gönderdim.

**Reply by ColdWind53** [reply]
> Discord üzerinden size yardımcı olabilirim. LadySmith#763 üzerinden size istek gönderdim.

Merhabalar güncel discord adınızı öğrenebilir miyim? birkaç sorum olacak.
